sphinx_immaterial/custom_admonition.py

```diff
@@ -0,0 +1,46 @@
+"""This module inherits from the generic ``admonition`` directive and makes the
```

Comment:
Perhaps rename this file to `md_admonition` to be consistent with the directive name?